docs/pom.xml

```diff
@@ -35,6 +35,20 @@
         <maven.install.skip>true</maven.install.skip>
         <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
     </properties>
+    
+    <dependencies>
+        <dependency>
+            <groupId>com.github.jobservice</groupId>
+            <artifactId>job-service-contract</artifactId>
+            <version>${project.version}</version>
```

Comment:
@tonymcveigh, I don't have a problem with this but did you tell me there was a problem with it?

Reply:
Using project.version in the dependency section is okay. Using it dependency management can results in problems

Reply:
Thanks build machine.  BTW have you become self-aware?